CHANGELOG.md

```diff
@@ -2,11 +2,14 @@
 
 ## 2.5 - ???
 
+### Enhancements
```

Comment:
I would consider this an addition which is a section we have sometimes when something totally new is added.